lib/metrics.go

```diff
@@ -73,6 +74,11 @@ func (m *Metrics) Add(r *Result) {
 		m.success++
 	}
 
+	if r.Code >= 400 && r.Code < 600 {
+		var codeWithURL string = "Code: " + strconv.Itoa(int(r.Code)) + " - " + r.URL
+		m.RequestsErrors = append(m.RequestsErrors, codeWithURL)
```

Comment:
This is a potentially huge slice. We could instead group the status code map by URL. I'm gonna experiment with that.

Reply:
OK, I don't think this is solid enough since there are many other request components that we could group by, like the request body, unique set of headers, etc. This is too complex to tack on to this report. You can do your own aggregation with `vegeta encode` and `jq` or `gron`.